File: tugas 823/tugas primaid/segitiga bintang (2).cpp
#include <iostream>
using namespace std;

int main() {
    int rows = 5;
    do {
        int j = 1;
        while (j <= rows) {
            cout << " ";
            j++;
        }
        int k = 1;
        while (k <= (5 - (rows - 1))) {
            cout << "*";
            k++;
        }
        cout << endl;
        rows--;
    } while (rows > 0);
    return 0;
}

File: tugas 823/tugas primaid/segitiga bintang (2).js
let rows = 5;

do {
    let line = "";
    let j = 1;
    while (j <= rows) {
        line += " ";
        j++;
    }
    let k = 1;
    while (k <= (5 - (rows - 1))) {
        line += "*";
        k++;
    }
    console.log(line);
    rows--;
} while (rows > 0);
